build/bin/vlan-filtering

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+vlan_range=1-4094
```

Comment:
There's one thing that puzzles me with the selected range.

So you activate vlan filtering **and** whitelist **all** the vlans (on all ports) ? It's unclear to me what are the intentions.

One possibility is that there are more "users" of that bridge and you don't want to impair their networking, but still, would like to point this out.

Reply:
We need to allow all incoming traffic from the host interface. When we configure the bridge we have no idea which NetAttDef are going to be defined in the cluster (nor do we want to know)